x/marker/client/cli/tx.go

```diff
@@ -544,9 +546,9 @@ func GetNewTransferCmd() *cobra.Command {
 	return cmd
 }
 
-// TODO: refactor usage comments to be provenance specific
 // GetIbcTransferTxCmd returns the command to create a GetIbcTransferTxCmd transaction
 func GetIbcTransferTxCmd() *cobra.Command {
+	// TODO: refactor ibc-transfer usage comments to be provenance specific
```

Comment:
Should this just be an issue?

Reply:
I'm not even sure what the TODO means. I just know that where it was, wasn't the correct place according to godoc standards. Godoc standards say the comment block before a function should start with the function name. The option was to either swap the two comment lines or move the TODO into the function. I chose the latter since it's not information that's worthwhile to have in the function's godoc/intellisense/info.

Reply:
Whoops, I thought it was just added.